src/diffusers/hooks/hooks.py

```diff
@@ -192,7 +192,7 @@ def remove_hook(self, name: str, recurse: bool = True) -> None:
             else:
                 self._fn_refs[index + 1].forward = old_forward
 
-            self._module_ref = hook.deinitalize_hook(self._module_ref)
+            self._module_ref = hook.deinitialize_hook(self._module_ref)
```

Comment:
```suggestion
            self._module_ref = hook.deinitalize_hook(self._module_ref)
```

Same reason as above